presto-orc/src/main/java/com/facebook/presto/orc/metadata/ColumnEncoding.java

```diff
@@ -46,12 +63,31 @@ public int getDictionarySize()
         return dictionarySize;
     }
 
+    public Optional<List<DwrfSequenceEncoding>> getSequenceEncodings()
```

Comment:
offline feedback: rename to something map specific

Reply:
@kevinwilfong : Actually @haozhun convinced me to use the current name. But maybe call it `getAdditionalSequenceEncodings`